Site/dataobjects/SiteAttachment.php

```diff
@@ -1,556 +1,448 @@
+     * Content-Disposition HTTP header.
+     *
+     * @var string
+     */
+    public $human_filename;
+
+    /**
+     * Mime type.
+     *
+     * @var string
+     */
+    public $mime_type;
+
+    /**
+     * File size of the attachment in bytes.
+     *
+     * Database field in numeric(10,0) since our systems can't support bigint
+     * due to be 32bit.
+     *
+     * @var float
+     */
+    public $file_size;
+
+    /**
+     * Whether or not this attachment has been copied to the CDN.
+     *
+     * @var bool
+     */
+    public $on_cdn;
+
+    /**
+     * The date that this attachment was created.
+     *
+     * @var SwatDate
+     */
+    public $createdate;
+
+    /**
+     * @var string
+     *
+     * @see SiteAttachment::setCDNBase()
+     */
+    protected static $cdn_base;
+
+    /**
+     * TODO.
+     *
+     * @var string
+     */
+    protected $attachment_set_shortname;
+
+    /**
+     * TODO.
+     *
+     * @var string
+     */
+    protected $file_base;
+
+    protected function init()
+    {
+        $this->table = 'Attachment';
+        $this->id_field = 'integer:id';
+
+        $this->registerDateProperty('createdate');
+
+        $this->registerInternalProperty(
+            'attachment_set',
+            SwatDBClassMap::get(SiteAttachmentSet::class)
+        );
+    }
+
+    public static function setCDNBase($base)
+    {
+        self::$cdn_base = $base;
+    }
+
+    public function getFormattedFileSize()
+    {
+        return SwatString::byteFormat($this->file_size, -1, false, 1);
+    }
+
+    public function getValidMimeTypes()
+    {
+        $ms_openxml_prefix = 'application/vnd.openxmlformats-officedocument';
+
+        return ['audio/mp4', 'video/mp4', 'audio/mpeg', 'application/zip', 'application/pdf', 'image/jpeg', 'image/png', 'text/html', 'application/msword', 'application/vnd.ms-excel', $ms_openxml_prefix . '.wordprocessingml.document', $ms_openxml_prefix . '.spreadsheetml.sheet'];
+    }
+
+    /**
+     * Returns the extension of the attachment based on  mime type.
+     *
+     * For MPEG-4 Audio, we use the non-standard but universally accepted m4a
+     * extension. See wikipedia for more details
+     * {@link https://en.wikipedia.org/wiki/.m4a}.
+     *
+     * @returns string The extension of the file.
+     *
+     * @param mixed|null $mime_type
+     */
+    public function getExtension($mime_type = null)
+    {
+        if ($mime_type == '') {
+            $mime_type = $this->mime_type;
+        }
+
+        $ms_openxml_prefix = 'application/vnd.openxmlformats-officedocument';
+
+        $map = ['audio/mp4' => 'm4a', 'video/mp4' => 'mp4', 'audio/mpeg' => 'mp3', 'application/zip' => 'zip', 'application/pdf' => 'pdf', 'image/jpeg' => 'jpg', 'image/png' => 'png', 'text/html' => 'html', 'application/msword' => 'doc', 'application/vnd.ms-excel' => 'xls', $ms_openxml_prefix . '.wordprocessingml.document' => 'docx', $ms_openxml_prefix . '.spreadsheetml.sheet' => 'xlsx'];
```

Comment:
PHPCS should be adjusted to not inline all arrays. This is much more difficult to read.

Reply:
I think what happened here is Rector converted the old array syntax to the new array syntax, but Rector has no sense of the original style, so it just outputs it all inline.  PHPCSFixer then doesn't expand single-line arrays by default ... but I suspect that can be changed.